cypress/e2e/davaleba12/test12.cy.js

```diff
@@ -0,0 +1,44 @@
+describe("automationexercise test cases", () => {
+    it("user registration without some informations", ()=>{
+cy.visit("https://automationteststore.com/")
+cy.get("#customer_menu_top > li > a").click()
+cy.get("#accountFrm > fieldset > .btn").click()
+cy.get("#AccountFrm_firstname").type('tamari')
+cy.get('#AccountFrm_lastname').type('eristavi')
+cy.get('#AccountFrm_address_1').type('bakhtrioni')
+cy.get('#AccountFrm_zone_id').select('Angus')
+cy.get('#AccountFrm_country_id').select('Georgia')
+cy.get('#AccountFrm_loginname').type('tamariii98')
+cy.get('#AccountFrm_password').type('paroli')
+cy.get('#AccountFrm_confirm').type('paroli')
+cy.get('#AccountFrm_newsletter0').check()
+cy.get('#AccountFrm_agree').check()
+cy.get('.col-md-2 > .btn').click()
+cy.get('.alert').should('be.visible')
+})
+
+it('user successful registration', () =>{
+    cy.visit("https://automationteststore.com/")
+    cy.get("#customer_menu_top > li > a").click()
+    cy.get("#accountFrm > fieldset > .btn").click()
+    cy.get('#AccountFrm_firstname').type('tamariko')
+    cy.get('#AccountFrm_lastname').type('eristavi')
+    cy.get('#AccountFrm_email').type('t.eristav99998@gmail.com')
+    cy.get('#AccountFrm_address_1').type('bakhtrioni')
+    cy.get('#AccountFrm_city').type('tbilisi')
+    cy.get('#AccountFrm_zone_id').select('Angus')
+    cy.get('#AccountFrm_postcode').type(101)
+    cy.get('#AccountFrm_country_id').select('United Kingdom')
+    cy.get('#AccountFrm_loginname').type('tamariiii98')
+    cy.get('#AccountFrm_password').type('paroli')
+    cy.get('#AccountFrm_confirm').type('paroli')
+    cy.get('#AccountFrm_newsletter0').check()
+    cy.get('#AccountFrm_agree').check()
+    cy.get('.col-md-2 > .btn').click()
+    cy.get('.heading2 > span').should('be.visible')
+    cy.get('#customer_menu_top > :nth-child(1) > .top > .menu_text').contains('tamariko')
+    cy.get("#topnav > .form-control").select("Specials")
```

Comment:
და ცალკე it ში უნდა იყოს , რეგისტრაციის ნაწილი არ არის Special Offers ის შემოწმება